app/javascript/map.js

```diff
@@ -0,0 +1,46 @@
+function initMap() {
```

Comment:
@lime666 https://github.com/lime666/google-map-app/pull/10#discussion_r849365564
same for the function names. here it could be `initDashboardMap` (~~initMap~~) and `initLocationPageMap` (~~initMap1~~)